pkg/route/api/allocator/allocator.go

```diff
@@ -0,0 +1,30 @@
+package allocator
```

Comment:
Yes


> On Mar 10, 2015, at 9:03 AM, Paul <notifications@github.com> wrote:
> 
> In pkg/route/api/allocator/allocator.go:
> 
> > @@ -0,0 +1,30 @@
> > +package allocator
> @smarterclayton is this something you think warrants an interface so it can have different implementations in the future (perhaps for different allocation strategies)?
> 
> —
> Reply to this email directly or view it on GitHub.
>